README.md

```diff
@@ -1,3 +1,69 @@
 # panther_ros
 
-Packages composing basic funcinalities of the Husarion Panther robot.
\ No newline at end of file
+Packages composing basic functionalities of the Husarion Panther robot.
+API for each node can be found in each package folder.
+
+## Build and run on hardware
+
+In order to build the Panther for hardware setup on internal computer, use following commands.
+``` bash
+export HUSARION_ROS_BUILD_TYPE=hardware
+
+pip3 install apa102-pi rosdep vcstool
+
+git clone https://github.com/husarion/panther_ros.git src/panther_ros
+vcs import src < src/panther_ros/panther/panther.repos
+rm -r src/panther_ros/panther_gazebo
+rosdep init
+rosdep update --rosdistro $ROS_DISTRO
+rosdep install --from-paths src -y -i
+source /opt/ros/$ROS_DISTRO/setup.bash
+catkin_make -DCATKIN_ENABLE_TESTING=0 -DCMAKE_BUILD_TYPE=Release
+```
+
+After successful build run
+``` bash
+roslaunch panther_bringup bringup.launch
+```
+
+This will launch all nodes related to Panther robot, selecting them to match your specific hardware revision. Please keep in mind, this packages require environment variables that are set during boot procedure of Panther.
+
+## Build and run simulation
+
+To build your system run:
+``` bash
+export HUSARION_ROS_BUILD_TYPE=simulation
+
+pip3 install rosdep vcstool
+
+git clone https://github.com/husarion/panther_ros.git src/panther_ros
+vcs import src < src/panther_ros/panther/panther.repos
```

Comment:
Don't we want to remove unnecessary packages? The fastest way will be something like this I guess:
```
mv panther_ros/panther_description/ src/panther_description
mv panther_ros/panther_gazebo/ src/panther_gazebo
rm -rf panther_ros
```

Reply:
Done